contracts/ChildChainGaugeInjectorV2.sol

```diff
@@ -355,10 +355,11 @@ KeeperCompatibleInterface
 /**
 * @notice Gets all current schedule information as a set of arrays
 */
-    function getFullSchedule() public view returns (address[] memory, uint256[] memory, uint8[] memory, uint56[] memory, uint56[] memory) {
+    function getFullSchedule() external view returns (address[] memory, uint256[] memory, uint8[] memory, uint56[] memory, uint56[] memory) {
```

Comment:
it's missing to return another item here; only returning 5 items, but expecting 6

Reply:
thanks.  Test wasn't running and env vars still not working so hardhat also didn't work.  Will do a final check of everything, run tests, think about that once we get this all merged in.